pkg/ossm/smcp_annotation.go

```diff
@@ -90,17 +90,35 @@ func TestSMCPAnnotations(t *testing.T) {
 		}
 		util.CheckPodRunning("bookinfo", "app=env")
 		time.Sleep(time.Duration(10) * time.Second)
-
-		msg, _ := util.Shell(`kubectl get po -n bookinfo -l app=env -o yaml | grep "annotation-from-smcp"`)
-
-		if !strings.Contains(msg, `test1.annotation-from-smcp: test1`) {
+		podName, _ := util.GetPodName("bookinfo", "app=env")
+		annotations, err := util.GetPodAnnotations("bookinfo", podName, 30)
+		if err != nil {
+			t.Errorf("Failed to get annotations: %v", err)
+			t.Fatal(err)
+		}
+		util.Log.Infof("Checking annotation: %", annotations["test1.annotation-from-smcp"])
+		if _, ok := annotations["test1.annotation-from-smcp"]; !ok {
 			t.Errorf("Failed to get annotations: %v", `test1.annotation-from-smcp: test1`)
+		} else {
+			if annotations["test1.annotation-from-smcp"] != "test1" {
+				t.Errorf("Failed to get annotations: %v", `test1.annotation-from-smcp: test1`)
```

Comment:
You don't need this placeholder when you don't print a variable:
```suggestion
				t.Errorf("Failed to get annotations: test1.annotation-from-smcp: test1")
```

Reply:
OK